controllers/perconaservermysql_controller.go

```diff
@@ -610,6 +611,10 @@ func (r *PerconaServerMySQLReconciler) reconcileOrchestrator(ctx context.Context
 		return nil
 	}
 
+	if cr.OrchestratorSpec().Size == 0 {
+		l.Info("Orchestrator not specified.")
```

Comment:
I'm not sure if this log is needed. what do you think @an3l?

Reply:
Following the same practice for all CRs.